src/Storages/System/StorageSystemZooKeeperConnection.cpp

```diff
@@ -31,7 +31,7 @@ void StorageSystemZooKeeperConnection::fillData(MutableColumns & res_columns, Co
     res_columns[3]->insert(context->getZooKeeper()->getConnectedZooKeeperIndex());
     res_columns[4]->insert(context->getZooKeeperSessionUptime());
     res_columns[5]->insert(context->getZooKeeper()->expired());
-    res_columns[6]->insert(context->getZooKeeper()->getApiVersion());
```

Comment:
@antonio2368 Probably we can provide a list of features that can be used in a scope of this connection instead?